src/tests/java/htsjdk/samtools/SAMFileReaderTest.java

```diff
@@ -53,6 +53,23 @@ public void variousFormatReaderTest(final String inputFile) {
         return scenarios;
     }
 
+    // tests for CRAM indexing
+
+    @Test(dataProvider = "")
+    public void CRAMIndexTest(final String inputFile) {
+        final File input = new File(TEST_DATA_DIR, inputFile);
+        final SamReader reader = SamReaderFactory.makeDefault().open(input);
+        Assert.assertTrue(reader.hasIndex());
+    }
+
+    @DataProvider(name = "")
```

Comment:
You need to give your `DataProvider` a name here, and use that name in the `@Test` annotation above.